pyrefly/lib/commands/check.rs

```diff
@@ -570,15 +570,30 @@ impl Args {
             print_error_summary(&errors.shown, path_index);
         }
         let shown_errors_count = config_errors_count + errors.shown.len();
+        let ignored_errors_count = errors.disabled.len() + errors.suppressed.len();
+        let transitive_dependencies_count = transaction.module_count() - handles.len();
         timings.report_errors = report_errors_start.elapsed();
 
         info!(
-            "{} errors shown, {} errors ignored, {} modules, {} transitive dependencies, {} lines, took {timings}, peak memory {}",
+            "{} error{} shown, {} error{} ignored, {} module{}, {} transitive dependenc{}, {} line{}, took {timings}, peak memory {}",
             number_thousands(shown_errors_count),
-            number_thousands(errors.disabled.len() + errors.suppressed.len()),
+            if shown_errors_count == 1 { "" } else { "s" },
```

Comment:
let's use the `count` function we already wrote, for the simple cases where we just add an s: https://github.com/facebook/pyrefly/blob/d24d371c02729d410147e96eefadab11996edc00/pyrefly/lib/util/display.rs#L141

Reply:
We're currently showing the numbers with thousands commas (using `number_thousands`) but `count(18208, "lines")` returns "18208 lines" and we want "18,208 lines".

Shall we add `number_thousands` into `count`, something like this?

```diff
 /// Format the number and descriptor word with the right pluralization.
 /// E.g., count(2, "cat") -> "2 cats". Pluralizes by tacking on an "s".
 pub fn count(n: usize, descriptor: &str) -> String {
-    format!("{n} {descriptor}{}", if n == 1 { "" } else { "s" })
+    format!("{} {descriptor}{}", number_thousands(n), if n == 1 { "" } else { "s" })
 }
```

Reply:
Sure, that could work. Maybe we could also make `count()` handle the `y`/`ies` pattern